src/main/resources/avro/variantmethods.avdl

```diff
@@ -8,9 +8,10 @@ import idl "variants.avdl";
 /** This request maps to the body of `POST /variantsets/search` as JSON. */
 record SearchVariantSetsRequest {
   /**
-  The Dataset to search.
+  If specified, will restrict the query to variant sets within the
+  given datasets.
   */
-  string datasetId;
+  array<string> datasetIds = [];
```

Comment:
Why are we rolling this back? We're already supporting exactly one dataset in the server, so the change required to move to `datasetId` from `datasetIds` is trivial. The changes required to properly support multiple datasets would be massive.